hsmodels/schemas/fields.py

```diff
@@ -163,16 +163,16 @@ class Config:
         description="A string containing the path to the hydroshare profile",
         allow_mutation=False,
     )
-    identifiers: Dict[UserIdentifierType, AnyUrl] = Field(
-        default={},
-        title="Creator identifiers",
-        description="A dictionary containing identifier types and URL links to alternative identifiers for the creator",
+    ORCID: AnyUrl = Field(default=None, title="ORCID identifier", description="Identifier from https://orcid.org/")
+    google_scholar_id: AnyUrl = Field(
+        default=None, title="Google Scholar identifier", description="Identifier from https://scholar.google.co.id/"
+    )
+    research_gate_id: AnyUrl = Field(
+        default=None, title="Research Gate identifier", description="Identifier from https://www.researchgate.net/"
```

Comment:
HydroShare also allows "ResearcherID". That's not included here. But it doesn't look like it was in the model before either?